### DISCUSSION TITLE
Search in grid view not working

### DISCUSSION THREAD

**Discussion by hung292001** [pimcore/pimcore#17756] [Q&A / Support] [answered]
I have list item like this 

![image](https://github.com/user-attachments/assets/0ba48a67-4df6-4fad-821f-f63129c58f77)

but when i search it didn't work

![image](https://github.com/user-attachments/assets/4c208503-2bf5-45e8-bbad-431ce646bb70)

it should look like this 

![image](https://github.com/user-attachments/assets/5c0aa25c-789c-42c3-8e36-7b64fc20c2e3)

Can anyone help me?

**Comment by arulselvan008** [comment, marked as answer]
Hi,

I have tried this command, and now it's working for me **bin/console pimcore:search-backend-reindex** 

Ref to the document also -> https://pimcore.com/docs/platform/Pimcore/Deployment/Deployment_Tools/

**Reply by hung292001** [reply]
Tks for your help, it worked.
